docs/sidebars.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   examples: [
     "examples/introduction",
+    "examples/frontend",
+    "examples/backend",
```

Comment:
I would dedicate one page as overview for examples, one page to quickstarts (with subpages), and move the rest as mentioned in the issue.

We should also consider getting rid of the examples section, to make it easier to move around the different sections and explore zitadel.